stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -112,11 +112,6 @@ public String getBenchDescription() {
   @Override
   @SuppressFBWarnings("BC_UNCONFIRMED_CAST")
   public void prepare() throws Exception {
-    if (mBaseParameters.mCluster && mBaseParameters.mClusterLimit != 1) {
-      throw new IllegalArgumentException(String.format(
-          "%s is a single-node client IO stress test, so it cannot be run in cluster mode without"
```

Comment:
If this constraint is lifted, what if multiple clients try to write to the same file when the operation is set to `WRITE`?

Reply:
Add the mParameter.mId into the file path, so that each client will write into its own directory. 
This makes sense for the test where every thread of every client reads its own file. But when we want all clients read the same file, it would result in all threads of one client read the same file, but different files across different clients.
It would be tricky for all clients read the same file, because the client doesn't have all the workerIds, thus doesn't know all generated test files. It has to ask for all directories from the master. I'm more inclined to keep the way now. What do you think?

Reply:
I don't have a better idea so let's keep it that way.